packages/web/src/components/remove/remove-liquidity-select-list-item/RemoveLiquiditySelectListItem.tsx

```diff
@@ -101,6 +102,8 @@ const RemoveLiquiditySelectListItem: React.FC<RemoveLiquiditySelectListItemProps
         </Tooltip>
         <Badge text={feeStr} type={BADGE_TYPE.DARK_DEFAULT} />
       </div>
+      {/* <span className="liquidity-value-fake" ref={liquidityRef}>${lpPosition.position.balance.toLocaleString()}</span>
+      <span className="liquidity-value" >${!checkWidth ? convertLargePrice(lpPosition.position.balance.toString()) : lpPosition.position.balance.toLocaleString()}</span> */}
```

Comment:
Is this for responsiveness? 
I'll reference it in my work.

Reply:
Yep. but have some logic above has been removed